src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -25,6 +27,12 @@
     private static final String MESSAGE_SINCE_DATE_LATER_THAN_UNTIL_DATE =
             "\"Since Date\" cannot be later than \"Until Date\"";
 
+    private static final DateFormat REPORT_NAME_FORMAT = new SimpleDateFormat("yyyy-MM-dd-HH-mm-ss");
+
+    private static String generateReportName() {
+        return REPORT_NAME_FORMAT.format(new Date());
```

Comment:
Let's rename this to `generateDefaultReportName()` and `DEFAULT_REPORT_NAME_FORMAT`.